### PR TITLE
#2309 - Update layout for pathway reaction on canvas

### PR DESCRIPTION
**Pull request by even1024**
## Generic request
- [x] PR name follows the pattern `#1234 – issue name`
- [x] branch name does not contain '#'
- [x] base branch (master or release/xx) is correct
- [x] PR is linked with the issue
- [x] task status changed to "Code review"
- [x] code follows product standards
- [x] regression tests updated

### CODE REVIEW

File: core/indigo-core/layout/pathway_layout.h [hunk 1]
```diff
@@ -42,18 +43,17 @@ namespace indigo
         DECL_ERROR;
 
     public:
-        PathwayLayout(PathwayReaction& reaction) : _reaction(reaction), _depths(10, 0), _maxDepth(0)
+        PathwayLayout(PathwayReaction& reaction, const LayoutOptions& options)
+            : _reaction(reaction), _depths(10, 0), _maxDepth(0), _bond_length(options.bondLength), _default_arrow_size((float)options.bondLength * 6.5f),
```
Comment (AlexanderSavelyev) [inline core/indigo-core/layout/pathway_layout.h:47]: add constant for 10

Comment (AlexanderSavelyev) [inline core/indigo-core/layout/pathway_layout.h:47]: add constant for  6.5

Comment (even1024) [reply core/indigo-core/layout/pathway_layout.h]: fixed

Comment (even1024) [reply core/indigo-core/layout/pathway_layout.h]: fixed

**Review by AlexanderSavelyev — APPROVED** [review]

**Pull request merged**